### PR TITLE
Add description for new feature on guide and feature introduction pages.

### PR DESCRIPTION
**Pull request by hallower**
Add description for new feature (http://tizen.org/feature/attach_panel)
to AttachPanel guide page and Tizen feature introduction page.

Signed-off-by: ChulSeung Kim <charles0.kim@samsung.com>

### CODE REVIEW

**Comment by Reni-90** [comment]
The files are reviewed and attached. Some of the hyperlinks were removed as they are redirecting to the same page, this can confuse user.
[Reviewed Files.zip](https://github.com/Samsung/tizen-docs/files/1859911/Reviewed.Files.zip)

**Comment by mijin85cho** [comment]
@Reni-90 Hello, Reni. There is one thing I want to share with you. 

Tizen Features are in `http://tizen.org/feature/blah-blah-` form, so it is often confused as an url. But, as you can see on https://www.tizen.org/feature, only just the form is in `http://tizen.org/feature` and it is not an actual url. The link just list up the features only for the reference.

So, below sentence can cause confusion. Please don't guide like below.
```
For more information, see (http://tizen.org/feature/attach_panel)
```

The **`http://tizen.org/feature/attach_panel`** itself is the `attach panel feature`. It is not a reference link.

**Comment by mijin85cho** [comment]
@Reni-90 
Hello.
As the way tizen feature is treated in reviewed file is not proper, would you please update it? The way http://www.tizen.org/feature/attach_panel is described would strengthen the misunderstanding.
I hope other pages describing features can be examples how to treat them, for example pages like https://github.com/Samsung/tizen-docs/blob/master/docs/application/native/tutorials/details/app-filtering.md or https://github.com/Samsung/tizen-docs/blob/master/docs/application/native/guides/device/system.md. Please refer them.

I'm also checking the attached files now. When I find any issue, I'll report here.

**Comment by Reni-90** [comment]
@mijin85cho 
Hello,
Sure, I will update the changes and attach the new files here.
In Tizen Features page, the format used is confusing. Please see the link, https://www.tizen.org/feature. All the headings are given in the format http://, I think this format must be changed to avoid confusions, if possible.

**Comment by Reni-90** [comment]
[Updated Files.zip](https://github.com/Samsung/tizen-docs/files/1878968/Updated.Files.zip)

The files are updated and attached.

**Comment by Reni-90** [comment]
@mijin85cho 
The latest updated files.
[Updated File.zip](https://github.com/Samsung/tizen-docs/files/1879703/Updated.File.zip)

**Comment by mijin85cho** [comment]
@Reni-90 Hello, Reni.
As I'm saying again and again, `https://www.tizen.org/feature/attach_panel` is a feature itself, 
so below suggested sentence is quite weird as a simple sentence and broken part of the sample code are mixed.
```
Tizen mobile devices supports the API that supports attach panel feature, `feature name = https://www.tizen.org/feature/attach_panel`.
```

I sent you a sample link, https://github.com/Samsung/tizen-docs/blob/master/docs/application/dotnet/guides/connectivity/nsd.md#prerequisites, hoping it can be a guide to you. 

Please study that page.

As you can see, in the **Prerequisites** part, where guide what to pre-set before a user start to develop a function. In the second part, you can see it is guiding how to specify the feature in tizen-manifest.xml file. I think this way would be better to guide `https://www.tizen.org/feature/attach_panel` in attach-panel.md.
Please remove `feature name = https://www.tizen.org/feature/attach_panel` from your suggested sentence, and add another sub part under  **Prerequisites** to explain required feature,  `https://www.tizen.org/feature/attach_panel`.  I think making it as a second part, after description for privilege would be better. Of course other original sub parts should be re-ordered.

Also, as http://tizen.org/feature/attach_panel is newly added feature for 5.0, as this PR is for tizen_5.0_prepare branch, it is not listed on http://tizen.org/feature. When Tizen 5.0 is published, it will added in that page.

@annie-samsung 
Would you please cross check Reni's reviewed file if possible? 
Thanks in advance.

**Comment by Reni-90** [comment]
[Updated File.zip](https://github.com/Samsung/tizen-docs/files/1882855/Updated.File.zip)
The files are updated as per the suggestion and attached.

**Comment by Reni-90** [comment]
[Updated File.zip](https://github.com/Samsung/tizen-docs/files/1883747/Updated.File.zip)
The latest updated files.

**Comment by Reni-90** [comment]
@mijin85cho 
Hello,
Updated the files for both native and dotnet. Changed the example to:
<feature name="http://tizen.org/feature/attach_panel"/>

[Updated File.zip](https://github.com/Samsung/tizen-docs/files/1892832/Updated.File.zip)

**Comment by hallower** [comment]
Dear @mijin85cho and @Reni-90, 
I updated AttachPanel guides by following revised files.
I reviewed the revised contents and there is no doubt that it is deserved to be merged.
Thank you.

**Comment by mijin85cho** [comment]
Dear @Reni-90 and @hallower 
Thank you for the Review, cross-check and Update. I approved this PR and merged it.

File: docs/application/native/guides/notification/attach-panel.md [hunk 1]
```diff
@@ -123,7 +134,7 @@ To create an attach panel:
     s_info.attach_panel = attach_panel;
     ```
 
-2. Based on the type of content you want the user to be able to select for the attach panel, add content categories using the `attach_panel_add_content_category()` function. The available content categories are defined in the [attach_panel_content_category](../../api/mobile/latest/group__CAPI__PANEL__ATTACH__MODULE.html#gada3a2db6ac8e7d42b7dff7c3cc48720b)enumerator.The content categories in the **More** tab are shown in the frequency, recently used, and alphabetical sequence.To deliver more information to the UI gadget or called application, add the data with a bundle.
+2. Select the type of content for the attach panel. Add content categories using the `attach_panel_add_content_category()` function. The available content categories are defined in the [attach_panel_content_category](../../api/mobile/latest/group__CAPI__PANEL__ATTACH__MODULE.html#gada3a2db6ac8e7d42b7dff7c3cc48720b) enumerator. The content categories in the **More** tab are shown in the frequency of recently used and alphabetical sequence. To deliver more information to the UI gadget or called application, add the data within a bundle.
```
Comment (Reni-90) [inline docs/application/native/guides/notification/attach-panel.md]: @hallower 
Hello,
 According to me,  [attach_panel_content_category] enumeration is a collection of different enumerators like ATTACH_PANEL_CONTENT_CATEGORY_IMAGE, ATTACH_PANEL_CONTENT_CATEGORY_CAMERA, ATTACH_PANEL_CONTENT_CATEGORY_VOICE, and so on. In that case, Please change the word enumerator to enumeration.

File: docs/application/native/guides/notification/attach-panel.md [hunk 2]
```diff
@@ -149,8 +160,8 @@ To create an attach panel:
...
-   - To get the published events from the panel side, register a callback using the `attach_panel_set_event_cb()` function. The callback is triggered when reserved events (defined in the [attach_panel_event](../../api/mobile/latest/group__CAPI__PANEL__ATTACH__MODULE.html#ga722a6d81e76fc1c4567a1bf920b4da3e) enumerator) are published from the panel side.
+   - To get the data that you select in the called application, register a callback using the `attach_panel_set_result_cb()` function. Select and confirm the content category for the caller application to trigger the event. When you use this callback, you must use the `app_control_get_extra_data_array()` function to get the received data.
+   - To get the published events from the panel side, register a callback using the `attach_panel_set_event_cb()` function. Publish the reserved events (defined in the [attach_panel_event](../../api/mobile/latest/group__CAPI__PANEL__ATTACH__MODULE.html#ga722a6d81e76fc1c4567a1bf920b4da3e) enumerator) from the panel side to trigger the event.
```
Comment (Reni-90) [inline docs/application/native/guides/notification/attach-panel.md]: Please verify and change enumerator to enumeration.

File: docs/application/native/guides/notification/attach-panel.md (hunk 1 above)
Comment (hallower) [reply docs/application/native/guides/notification/attach-panel.md]: Thank you for comment.
Would you let me know what this mean "Please change the word enumerator to enumeration."?

Comment (Reni-90) [reply docs/application/native/guides/notification/attach-panel.md]: The available content categories are defined in the [attach_panel_content_category](../../api/mobile/latest/group__CAPI__PANEL__ATTACH__MODULE.html#gada3a2db6ac8e7d42b7dff7c3cc48720b) enumerator. 
In the sentence, attach_panel_content_category is an enumeration, but in the sentence it is written as enumerator.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

File: docs/application/native/guides/notification/attach-panel.md (hunk 1 above)
Comment (hallower) [reply docs/application/native/guides/notification/attach-panel.md]: Dear @Reni-90 
I will raise new pull request for that.

**Pull request merged**